spec/factories/reviews.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :review do
+    book nil
+    user nil
+    body "review body text"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.